x/posts/internal/types/msgs.go

```diff
@@ -134,7 +134,7 @@ func (msg MsgLikePost) Type() string { return ActionLikePost }
 // ValidateBasic runs stateless checks on the message
 func (msg MsgLikePost) ValidateBasic() sdk.Error {
 	if !msg.PostID.Valid() {
-		return sdk.ErrUnknownRequest("Invalid post id")
+		return sdk.ErrUnknownRequest("Invalid post newID")
```

Comment:
I think this is a refactoring error

```suggestion
		return sdk.ErrUnknownRequest("Invalid post id")
```